benchmarks/defaults.hpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <key_generator.hpp>
+
+#include <nvbench/nvbench.cuh>
+
+#include <cstdint>
+#include <vector>
+
+namespace cuco {
+namespace benchmark {
+
+namespace defaults {
+using KEY_TYPE_RANGE   = nvbench::type_list<nvbench::int32_t, nvbench::int64_t>;
+using VALUE_TYPE_RANGE = nvbench::type_list<nvbench::int32_t, nvbench::int64_t>;
+
+auto constexpr N             = 100'000'000;
+auto constexpr OCCUPANCY     = 0.8;
```

Comment:
I'm inclined to use `0.5` as the default occupancy since it's also the default occupancy in libcudf. It happened several times that we improved performance for a certain occupancy but introduced performance regression in libcudf.

Reply:
addressed in [ccf10d9](https://github.com/NVIDIA/cuCollections/pull/271/commits/ccf10d9da10462cbb51b320c6f77fad15332d04d)